FlightsWebsite/src/js/flights.js

```diff
@@ -1,14 +1,112 @@
-
+//import Converter from "./monthConventer";
 const url = 'https://flights-service-buki55n7ba-lz.a.run.app/flights/';
 
-function getFlights() {
+function getTotalFlights() {
 
     fetch(url + 'getTotalNumberOfFlights')
         .then(status)
         .then(json)
         .then(function (data) {
-            // document.getElementById("myText").innerHTML = responseData;
-            console.log(data);
+
+            let result = [];
+            data.forEach(element => {
+                    result.push({y: element.countOfFlights, label: convertMonth(element.month)});
+                }
+            )
+
+            showTotalFlights(result)
+
+        }).catch(function (error) {
+        console.log('Request failed', error);
+    });
+}
+
+async function getFlightsFromOrigins() {
```

Comment:
Can you change that chart so it is more clear? I would remove color under the plot and change the starting point and interval, so that the data for LGA and EWR are not overlaping